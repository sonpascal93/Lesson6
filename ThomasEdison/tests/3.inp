20
-.
57
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
--.---.-.--..-.-.-..--........-..-..--..-..-.-.-.--.-.--.--.....--.-.---.-...-..-.-..-...-.--.
97
GORZNCWIHIXIZNRCG
KYISQJA
IL
NXV
P
GORZNCWIH
IXIZNRCGKYISQ
JAIL
NXVP
GORZNCWIHIXIZNRCGKYISQJA
I
LN
XVP
GORZNCWIHIXIZNRCGKYISQJAILNXV
P
GORZNCWI
HIXIZNRCGKYIS
QJAI
LNX
V
P
GORZNCWIH
IXIZNRCGKYISQJAILNXVP
GORZNCWIHIXIZNRCGKY
ISQJAILNXVP
GORZNCWIHIXIZNRCGKYISQ
J
AILNXVP
GORZNCWIHIX
IZNR
CGKYISQ
JAILNX
V
P
GORZN
CWIHIXI
ZNRCGKYISQJAIL
NX
V
P
GORZNCWIHIXIZNRCGKYISQJAILN
XVP
GORZNCWIHIXIZNRCGKYIS
QJAILN
X
V
P
GORZNCWIHIXIZNRCGKYISQJAILNXVP
GORZNCWIHIXIZ
NRCGKYISQJA
ILN
X
V
P
GORZNCWIHIXI
ZN
RCGKYIS
QJAILN
X
VP
GORZNCWIHIXIZNRCGKYISQJAILNX
VP
GORZNCWIHIXIZNRCGKYISQJAILNX
V
P
GORZNCWIHIXIZNRCGKYISQJAIL
NX
VP
GORZNCWIHIXIZNRCGK
YISQJAILN
X
V
P
GORZNCWIHIXIZNRCGKY
I
SQ
J
A
ILNXV
P
GORZNCWIHIXIZNRCGKY
IS
QJAILNX
VP
GORZNCWIHIXIZNRCGKYISQJAILNXVP
GORZNCWIHIXI
ZNRC
GKYI
SQJ
AILNXVP
GORZNCWIHIXIZNRCGKYISQJ
AILNX
V
P
GORZN
CWIHIXIZNR
CGKYISQJ
-----.-......-...-.-...-.-.--.-..-...----.-...---.--...-------.-..-.--.-...-..-.--...--....-..-.----.-...-...-----....-.---.-..-.....--....---....-..-.--..----..--..--...---......--.....-.-.-.--..---...--..-..-.--.....--.--
34
OQSVEFNFKCRUOLJANUOMKUWLLTWSGSX
WQVVMZUJRLVBJHNRZMZGWVZHGHCKXZEZDYHPM
OQSVEFNFKCRUOLJANUOMKUWLLTW
SGSXWQVVMZUJRLVBJHN
RZMZGWVZHGHCKXZEZDY
HP
M
OQSVEFNFKCRUOLJANUOMKUWLLTWSGSXWQVVMZUJRLVBJHNRZMZGWVZHGHCKXZEZ
DYHP
M
OQSVEFNFKCRUOLJANUOMKUWLLTWSGSXWQVVMZUJRLVBJHNRZMZ
GWVZHGHCKX
ZEZDYHP
M
OQSVEFNFKCRUOLJANUOMKUWLLTWSGSXWQVVMZUJRLVBJHNR
ZMZGWVZH
GHCKXZEZDYHPM
OQSVEFNFKCRUOLJANUOMKUWLLTWSGSXWQVVMZUJRLVBJHNRZMZGWVZHGHCKXZE
ZDYHP
M
OQSVEFNFKCRUOLJANUOMKUWLLTWSGSX
WQVVMZUJRLVBJHNRZMZGWVZHGH
CKXZEZD
YHP
M
OQSVEFNFKCRUOLJANUOMKUWLLTWSGSXWQVVMZUJRLVBJHNRZMZGW
VZHGH
CKXZEZDYHP
M
OQSVEFNFKCRUOLJANUOMKUWLLTWSGSXWQVVMZUJRLVBJHNRZ
MZGWVZHGHCKXZEZDYHPM
OQSVEFNFKCRUOLJANUOMKUWLLTWSGSXWQVVMZ
UJRLVBJHNRZMZG
WVZHGHCK
--....---..--.--.----.-.--.---...-.....-.-...--.--.....-
11
ZU
ZM
PON
NM
WBBULP
ZV
Z
UZMPONNM
WBB
ULP
Z
-.--..-.-.-...-...-....-.--.-...-..--.----...-.....-.--.-..---.-.----...-....-.....-.--.-.---..--.---.--..---.-.---...---.-.-........-...-.-..-----.-..-..--.
66
YUEKSBNVPBDQOVSUPDMNNONIDUS
FGYXYKXQJ
VQRIHLECEOGTF
P
YUEKSBNVPBDQOVSUPDMNNONIDUSFGYXYKXQJVQRIHLECEOGTFP
YUEKSBNVPBDQOV
SUPDMNNONIDUSFGYXYKXQJVQRIHLEC
EOGTF
P
YUEKSBNVPBD
QOVSUPDMNNONIDUSFGY
XYKXQJVQRIHL
ECEOGTFP
YUEKSBNVPBD
QOVSUPDMNNONIDUSFGYXYK
XQJVQRIHLECE
OGTF
P
YUEKSBNVPBDQOVSUPDMNNONIDUSFGYXYKXQJVQR
IHLECEOGT
F
P
YUEKSBNVPBDQOVSUPD
MNNONIDUSFGYXYKXQJVQRIHLE
CEO
GT
FP
YUEKSBNVPBDQOVSUPD
MNNO
NIDUSFGYXYKXQ
JVQRIHLE
CEOGTFP
YUEKSBNVPBDQOVSUPDMNNONIDUSFGYXYKXQJVQRIHLECEOGTF
P
YUEKSBNVP
BDQOVSUPDMNNONIDUSFGYXYKXQJVQ
RIHLECEO
GTF
P
YUEKSBNVPBDQOVSUPDMNNONIDUSFGYXYKXQJVQRIHLECEOGT
FP
YU
EKSBNVPBDQOVSUPDMNNONIDUSFGYXYKXQJVQR
IHLECEOGTFP
YUEKSBNVPBDQOVSUPDMNNONIDUSFGYXYKXQJVQR
IH
LE
C
EOGT
F
P
YUEKSBNVPBDQOVS
UPDMNNONIDUSFGYX
YKXQJVQRIHLE
C
EOG
T
F
P
YUEKSBNVPBDQOVSUPDMNNONIDUSFGYXYKXQJ
VQRIHLECEO
GTFP
YUEKSBNVPBDQOVSUPDMNNONIDUSFGYXYKXQJVQRIHLEC
E
OGTF
P
-...-.-.....--.-.-....-...-.-.-.-.-.....--.--...---.-----.-..-.-.-..-.-..--..--.--....--....-.--.-..-...--..-.-.-.-----..-......------..---.-.......--.-...--.-.----.-....-.-....-.-.-.---.--..-..-.-..-..--.--.......-..-.--.-..--..-.....--..-.-...--.....
64
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASR
THKROPRRNLQBHXW
RADDVTFD
WI
S
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHKROPRR
NLQBHXWRADDVTFDWIS
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHKROPRRNLQBHXWRADDVTFDW
IS
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQ
VKJTASRTHKROPRRNLQB
HXWRADDVTF
DWIS
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHKROPRRNLQBHXWRADDVT
FDWI
S
NICHGKHDRCKSEPZJJQ
ICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHK
ROPRRNLQBHXWRA
DDVTFDW
IS
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHKROPRRNLQ
B
HXWRADDV
TFDWIS
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTA
SRTHKROPRRNLQBHXWR
A
DDV
TFD
WI
S
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHKROPRRNLQBHXWRADDVTFDWIS
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHKROPRRNLQBHXWRADDV
TF
D
WIS
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMO
DTTNBHQVKJTASRTHKROPRRNLQBHXW
RAD
DVTFD
WI
S
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBM
UAROZBSMODTT
NBHQVKJTASRTHKROPRRNLQBHXW
RADDV
TFDWIS
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHKROPRRNLQBHXWRADDVTF
DW
IS
NICHGKHDRCKSEPZJJQI
CXLGPMHGVWLBMUAROZBSMODTTNB
HQVKJTASRT
HKROPRRNLQBH
XWRADD
VTFD
WIS
NICHGKHDRCKSEPZJJQICXLGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHKROPRRNLQB
HXWRADD
VTF
DWIS
NICHGKHDRCKSEPZJJQICX
LGPMHGVWLBMUAROZBSMODTTNBHQVKJTASRTHKROPR
-...---.---.--.-.--.-----.---.....-..-..---..--....-----......--..-..-...-..-..-.-...--.-.-..--.--....-.-...--...-.---..---....--....-.-...---.--..--.--.-..--.......--.--..--..-----....---.....-..-.-.-..-.--.....-.----.-.....-.......-..----.-.-...---.--....-..---....--......-
79
BMYTWRQOYDILXMEWSJ
GHPA
ELAUFBQRPGVLPUWXZEPS
CEJPWACA
BSPZZMOHMBUFKFZSKTQSUH
VUMKLAQ
DFJHG
SU
BMYTWRQOYDILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSCEJPWACABSPZZMOHMBUFKFZSKTQSUHVUMKLAQDF
JHGSU
BMYTWRQOY
DILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSCEJPWACABSPZZMOHMBUFKFZ
SKTQSUHVUMKLAQDFJHG
S
U
BMYTWRQOYDILXMEWSJGHPA
ELAUFBQRPGVLPUWXZEPSCEJPWACABSPZZMOHMBUFKFZSKTQSUHVUMKLAQDFJHGS
U
BMYTWRQOYDILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSCE
JPWACABSPZZ
MOHMBUFKFZSKTQSU
HVUMKLAQD
F
JH
GS
U
BMYTWRQOYDILXMEWSJGHPAELAUFBQRP
GVLPUWXZEPSCEJPWACABSPZZ
MOHMBUFKFZSKTQSUHVUMKLAQDFJ
HG
S
U
BMYTWRQOYDILXMEWSJGHPAELAUFBQRPGVLPUW
XZEPSCEJPWACABSPZZMOHMBUFKFZSKTQSUHVUMKLA
QDFJHGS
U
BMYTWRQOYDILXMEWSJGHPAEL
AUFBQR
PGVLPUWXZEPSCEJPWACABSPZZMOHMBUFKF
ZSKTQSUHVUMK
LAQDFJHGSU
BMYTWRQO
YDILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSCEJPWACABSPZZMOHMBUFKFZSKT
QSUHVUMKLAQD
FJHG
S
U
BMYTWRQOYDILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSCEJPWACAB
SPZZMOHMBUFKFZS
KTQSUH
VUMKLAQDFJHG
SU
BMYTWRQOYDILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSCEJPWACABSPZZMO
HMBUFKFZSKTQSUHVUMKLAQDFJ
HG
SU
BMYTWRQOYDILXMEWSJGHPAELAUFBQ
RPGVLPUWXZEP
SCEJPWACABSPZZMOHMBUFKFZSKTQSUHVUMKLAQDFJHGSU
BMYTWRQOYDILXMEWSJGHPAELAUFBQRPGVLPUWXZEP
SCEJPWACABSPZZMOHMBUFKFZSKTQSUHVUMKLAQDFJHGSU
BMYTWRQOYDILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSCEJPWACABSP
ZZMOHMBUFKFZSKTQSUHVUMKLA
Q
DFJHGSU
BMYTWRQOYDILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSC
EJPWA
CABSPZZMOH
MBUFKFZSKTQS
UHVUMKLAQDFJHG
SU
BMYT
WRQOYDILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSCEJPWACABSPZZMOHMB
UFKFZSKTQSUHVUMKLAQDF
JH
GSU
BMYTWRQOYDILXMEWSJGHPAELAUFBQRPGVLPUWXZEPSCEJPWACABSPZZMOHMBUFKFZSKTQSUHVUMKLAQ
DFJHG
S
----.---..-.-...-.-.-.-.---.-.....-.----...-.-.-.--.--...--....--.-.-
67
OYXRFCYCS
ROBCYP
UBWAE
T
OYXRFCYCSROB
CYPUBWAET
OYXRFCYCSROBCYPU
BWAE
T
OYXRFCYCSROBCYPU
BWAET
OYXRFCY
CSR
OB
CYPUBWAET
OYXRFCYCS
ROBCYPUBWAET
OYXRFCYCSROBCYPUBWA
E
T
OYXRFCYCSRO
BCYPU
BW
A
ET
OYXRFCYCSROBCYPUBWAET
OYX
RFCYC
S
ROBCYPUBW
AE
T
OYXRFC
YC
SROBCYPUBWAET
OYXRFCYCSROBC
YPUB
WAET
OYXRFCYCSROBCYPUB
WAET
OY
XR
FCYCSROBC
YPUB
WAET
OYXRFCYCSROBCYPUBWAET
OYXRFCYCSROBCYPUBWAET
OYXRFCYCSROBCYP
U
B
W
AE
T
OYXRFCYCSROBCYPUBWAET
OYXRFCYCSROBCYPUBWAE
T
OY
XR
FCYCSROBCYP
UBWAE
T
OYXRFCYCSROBCYPUBW
A
E
T
OYXRFCYCSROBCYPUBWA
E
..-......---.-..-----.--..-.-.--..-----.-.-..
68
FHWKUOYUEKXONCE
FHWKUOYUEK
XO
N
CE
FHWKUOYUEKXONC
E
FHWKUOYUEKXON
CE
FH
WKUOYU
EKXONC
E
FHWKUOY
U
EKXON
C
E
FHWKUOY
UEKXONCE
FHWKUOYU
EKXONCE
FH
WKUOYUEKXONC
E
FHWKU
OYUEKXONCE
FHWKU
O
YUEKXO
N
CE
FHWKUOYUEKXON
C
E
FHWKUOYUEKXONC
E
FHWKUOY
UEK
XON
C
E
FHWKUOY
UEKXONC
E
FHW
KUOYUEKXONCE
F
H
WK
UOYUEKXO
N
CE
FHW
KUOYUEKXON
CE
FHWKUOYUEK
X
ON
CE
FHWKUOYUEK
XONC
E
FH
WKUOYUEKX
O
NC
E
-----..-.--.----....-.-..-...-.-..--...--....-.-..-.-.-..-....---.---.----..-....--..-.-.--.--.-.-.-.-..-.--.--.....-.-.----.-..-.--..-...--.-.---.---.-....-...-.-..-...-..---.----..---.....--..--.-..--...--.--...-...-..-.-..--.-..-.---.--....-..---.-..--.----...-.----..-
34
OZYJBRDBCPUDULCTFVQQOUSADCQCKAFQNSRYMLYFIQWYCIEBCLRJATGWTHPPXDPZRFRXKFMYHXGXK
O
VJX
OZYJB
RDBCPUDULCTFVQQOUSADCQCKAFQNSRYMLYFIQWYCIEBCLRJATGWTHPPXDPZRFRXKFMYHX
GX
KOVJX
OZYJBRDBCPUDULCTFVQQOUSADCQCKAFQNSRYMLYFIQWYCIEBCLRJATGW
THPPXDPZRFRXKFMYH
XGXKOVJX
OZYJBRDBCPUDULCTFVQQOUSADCQCKAFQN
SRYMLYFIQWYCIE
BCLRJATGWTHPPXDPZRF
RXKFMYHXGXKOVJ
X
OZYJBRDBCPUDULCTFVQQOUSADCQCKAFQNSRYMLYFIQWYCIEBCL
RJ
A
TGWTHPPXDPZR
FRXK
FMYHXG
X
KOV
JX
OZYJBRDBCPUDULCTFVQQOUSADCQCK
AFQNSRYMLYFI
QWYCIEBCLRJATGWTHPPXDPZRFRXKFMYHXGXKO
VJ
X
OZYJBRDBCPUDULCTFVQQOUSADCQCKAFQNSR
Y
MLYFIQWYCIEBCLRJATGWTHPP
XDPZRFRX
KFMYHXGXKOV
.--........-.------.-.---..-.....----......-.--.-......--.--.--.--.--.-....-.--.--..--..-.--..-..-.---..----.--...--...-.-.--.-.--.-..--..--.-----.------.----.-.--.--....-.-...--..---..-.-.....--.--.-..
19
PHSYOKWXHWZHYRHWPQYLFQX
XPLYDOYVNUAWEYRWEPOGTMOJNYPSNDEZMDNB
U
Y
L
PHSYOKWXHWZHYRHWPQYLFQXXPLYDOYVN
UAWEYRWEPOGTMOJNYPSNDEZMDNBU
Y
L
PHSYOKWXHWZHY
RHWPQYLFQXXPLYDOYVNUA
WEYRWEPOGTMOJNYPSN
DEZMDNBUYL
PHSYOKWXHWZHYRHWPQYLFQXXPLYDOYVNUAWEYRWEPOGTMOJNY
PSND
EZMDNBUY
L
PHSYOKWXHWZHYRHWPQYLFQXXPLYDOYVNUAWEYRWEPOGTMOJNYPSNDEZMDN
BUY
.---..----..--....-...-.--.....-----.--.-....-.-..-..--..-...-.--.----..-...--..---...-..-.-.-.-.---..--..-.-.-.-.-.-.....-----.--.....-...--
40
WDOXBLAPHOQCVLXDBYJXVXGIXAEC
MDM
IK
RKL
I
JQBFIM
WDOXBLAPHO
QCVLXDBYJ
XV
XGIXAECMDMIKRKLI
JQB
FIM
WDOXBLAPHOQCVLXDBYJXVXGIXAECMDMIKRKLI
JQBFI
M
WDOXBLAPHOQCVL
XDBYJXVXGIX
AECMD
MIKRKLIJQBFIM
WDOXBLAPHOQCVLXDBYJXVXGIXAECMDMIKRKLIJ
QBFIM
WDOXBLAPHOQC
VLXDBYJXVXGIXAECMDMIKRK
LIJQBFI
M
WDOXBLAPHOQCVLXD
BYJXVXGIXAECMDMIKRKLIJQBF
I
M
WDOXBLAPHOQCVLXDBYJXVXGIXAECMDMIKRKLIJQ
B
FIM
WDOXBLAPHOQCVLXDBY
JXVXGIX
AECMDMIKRKLIJQBF
IM
WDOXBLAPHOQ
CVLXD
BYJXVXGIXAECMDMIKR
KLIJQBFIM
.--...-..--..-...---.-.--.-.----.-.----...--...--.--.-
7
PFEZBMKPTWG
YTBZ
PQ
PFE
ZBMKPTWGY
TB
Z
-.--.----.--...-.----.-.----.--...-..--.--.-....--.-..-.------...--...---..-..-.-...-..---.--.--.--.--.--.-.--..-.-.--.----..---.-...---.-..-.-..-....--.---.---.-..-.-.-....-
83
YAMYVJNTWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQ
VQUAFIPMYNTF
NB
A
YAMYVJNTWQBXYLIQEE
YOBZWXU
LRJPQNGQWURMJXQVQUA
FIPMYNTFNB
A
YAMYVJN
TWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQVQUA
FIPM
YNTF
NBA
YAMYVJNTWQB
XYLIQEEYOBZWX
ULRJPQN
GQWURMJXQVQUAF
IPMY
NTFNBA
YAMYVJNTWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQVQ
U
AFIPMYNTF
NBA
YA
MYVJNTWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQVQ
UAFIPMYNTFNB
A
YAMYVJNTWQBXYLIQEEYOBZWXULRJP
QNGQWURMJXQVQ
UAFIPMYNTF
NBA
YAMYVJNTWQBXYLIQEEYOBZWX
ULRJPQNG
QWURMJXQVQUAF
IPMY
NTFNBA
YAMYVJNTWQBXYLIQE
EYOBZWXULRJPQNGQWURMJXQVQUAFIP
M
YNTF
N
B
A
YAMYVJNTWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQVQUAFIPMY
NTFNB
A
YAMYVJNTWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQVQUAFIPMYNT
F
NBA
YAMYVJNTWQBXYLIQEEYO
BZWXULRJPQNGQWURMJXQVQ
UAFIPMYNTF
N
BA
YAMYVJNTWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQVQUAFIPMYN
TFNB
A
YAMYVJNTWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQVQUAFIPMYNTFNB
A
YAMYVJNTWQBXYLIQEEYO
BZWXULRJPQNGQWURMJXQ
VQUAFIP
MYNTFNB
A
YAMYVJNTWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQVQU
A
F
IPMYNTFNBA
YAMYVJNTWQBX
YLIQEEYOBZWXU
LRJPQNGQWURMJXQVQUAFIPMYNT
FNBA
YAMYVJNTWQBXYLIQEEYOBZWXULRJPQNGQWURMJXQVQUAFIPMYN
TF
NBA
YAMYVJNTWQBXYLIQEEYOBZWXULRJPQ
NGQWURMJXQVQUA
FIPMYNT
FNB
A
YAMYVJN
TWQB
--.-.--..--...--.--..-----.---..--....---..-..----.-...-...-.-.----.--.--.----.-.....-.-.-..-...-....-.-.-....-.--.......-----.-....-.--..-.-...-...--.----...----.-...-..-....--..--.-...-....-.....-..-----.-.-..-
92
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLL
UARSKTHIJQHYULASQOVORUFS
ZGDLUSF
AMQR
A
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLLUARSK
THIJQHYULASQOVORUFSZGDLUSF
AMQR
A
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLLUARSKTHIJQHYULASQOVORUFSZGDLUSFA
MQR
A
QWIZATPAOYXNVZNWQSDRTJY
EG
ONBEAECLLUARSKTHIJQH
YUL
ASQOVORUFSZGDL
USF
AMQRA
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLLUARSK
THIJQHYULASQOVORUFSZGDLU
S
FAMQRA
QWIZATPAOYXNVZNWQSDR
TJYEGON
BEAECLLUARSKTHIJQHYULASQOVORUFSZGDLUSFAMQR
A
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLLUARSKTHIJQHYULA
SQOVORUFSZGD
LUSFAMQRA
QWIZATPAOYX
NVZ
NWQSDRTJYEGONBEAECLLUARSKTHIJQHYULASQOVORUFSZG
DLUSFAM
QRA
QWIZATP
AOYXNVZNWQSDRTJYEGONBEAECLLUARSKTHIJQHYULASQOVORUFSZGDLUSF
AM
Q
R
A
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLLUARSKTHIJQHYULASQOVORUF
SZGDLUSFAMQ
R
A
QWIZATPAO
YXNVZNWQS
DRTJYEGONBEAECLLUARSKTHIJQHYULASQOVO
RUF
SZGD
LUSFAMQ
RA
QWIZATPAOYXNVZNWQSDRT
JYEGONBEAECLLUARSKTHIJQHYULASQOVORUF
SZGDL
USF
A
MQR
A
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLLUARSKTHIJQHYULASQOVORUFSZGD
LUS
FAMQRA
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLLUARSKTHIJQHYULASQOVORUFSZGDLUS
FAMQ
RA
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLLUARSKTHIJQHYULA
SQOVORUFSZGDLUS
FAMQRA
QWIZATPAOYXNVZNWQSDRTJYEGONBEAECLLUARSKTHIJQHYULASQOVORU
FSZ
GDL
U
S
FA
M
QR
A
QWIZATPAOYXNVZNWQSDRTJY
EGONBEAECLLUARSKTHIJQHYULASQOVORUFSZGDLU
SFA
MQ
R
A
QWIZA
TPAOYXNVZNWQSDRTJ
YEGONBEAECLLUA
RSKTHIJQHYU
LASQ
OVORUFSZGDLUS
FAMQR
A
QWIZATPAOYXNVZNWQSDRT
......-..--..--.---.-.-.---.---.--.----.-.--...--.--.-.-.-....-----.-.....-.-----.--.--..--.....-......--.-...-.-....--..-..-..---.--.-.-..---.-.----..--.....--.-.-.--.---..--....--.-..--...----........-..--.-.--.---.----.--.--...-....--..---...---..---.....-.--.-...-...-..--.--....-...-......
23
HIXXY
CYYYJCZPQAEBWTGNSROMPZGHBIWEBTRVNAFAQKLMNKMXNHQRQZZEWRPEWG
HVUCMJJKYVHZ
OSMXGH
YR
ELL
GGS
BB
S
HIXXYCYYYJCZPQAEBWTGNSROMPZGHBIWEBT
RVNAFAQKLMNKMXNHQRQZZ
EWRPEWGHVUCMJJKYVHZOSMXGHY
RELLGGS
BB
S
HIXXYCYYYJCZPQAEBWTGNSROMPZGHBIWEBTRVNAF
AQKLMNKMXNHQRQZZEWRP
EWGHVUCMJJKYVHZOSMXG
HYRELLGGSBBS
HIXXYCYYYJCZPQAEBWTGNSROMPZGHBIWEBTRVNAFAQKLMNKMXNHQRQZZEWRPEWGHVUCMJJKYVHZOS
MXGH
YRE
L
...-..-.-...--....-.--....-.--..---..---.-....---.....-..-....-.-.-....--...--...-.--..---....--.-.--.--.-.-....--.--.-..------..-..--.-..-....---.-...-.---.-....-.----.-.-.--....-..--.--......--.-----...---.....-.....---.--.-..--......-..-...-....-..-.---...-.--...-.--....--.----.....---...-..-.....---
35
IEXLPSYHKDOUGBAZILNIAALUBZAPJSPKKKLUKCWMMFPXHMCUW
CVWQRZFPZHQMGITGSLIJWLZHXSBRAJSKDAEZ
IGM
ZVTB
D
BU
M
IEXLPSYHKDOUGBAZILNIAALUBZAPJSPKKKLUKCWMMFPXHMCUWCVWQRZF
PZHQMGITGSLI
JWLZ
HXSB
R
AJSKDAEZIG
MZV
TBD
BU
M
IEXLPSYHKDOUGBAZILNIAALUBZAPJSPKKKLUKCWMMFPXHMCUWCVWQR
ZFPZHQ
MGITGSLIJWLZHXSBRAJS
K
DAE
ZIGMZVTBDBUM
IEXLPSYHKDOUGBAZILNIAALUBZAPJSPKKKLUKCWMMFPXHMCUWCVWQRZFPZ
HQMGITGSLIJWLZHXSBRAJSKDAEZ
IG
MZVTBDB
UM
IEXLPSYHKDOUGBAZILNIAALUBZAPJSPKKKLUKCWMMFPXHMCUWCVWQRZFPZHQMGITGSLIJWLZHXSBRAJSKDAEZ
IGMZVTBDB
UM
IEXLPSYHKDOUGBAZILNIAALUBZAPJSPKKKLUKCWMMFPXHMCUWCVWQRZFPZHQMGITGSLI
JWLZHXS
BRAJSKDAEZIG
M
.---.---..----.-..--...-.-.-.---...-..--.--.....--.....-.-..--..-.-..--....--.-----.--..---........-....--.--..--.-...-..-.-.-.---..-..--..---.--.-..-....-.---
24
JAZOCABCNOILGZVBFNWFXBWJMANEOSSERSGZQVFCOFADT
QNNL
UJ
JAZOCABCNOILGZVBFNWFXB
WJMANE
OSSERSGZQVFCOFADTQNNL
UJ
JAZOCABCNOILGZVBFNWFXBWJMA
NEOSSERSGZQV
FCOFADTQN
NL
U
J
JAZOCABCNOILG
ZVB
FNWFXBWJMANEOSSERSGZQVF
C
OFADTQNNL
UJ
JAZOCABCNOILGZVBFNWFXBWJMANEOSSERSGZQVF
COFADTQNN
LU
J
JAZO
..--.-...-.--.-.....-----.-------...-..-.-...-...---.....-.-----.-.-.-.....-.--.---.-..---....-.-...-...-.-..-
19
UKVWLV
MGOOBXLAVZV
WOECDVWJRJIFDRFX
UKVWLVMGOOBXLAVZVWOECDVWJRJI
FDRF
X
UKVWLVMGOOBXLA
VZVWO
ECD
VWJ
RJIFDRF
X
UKVWLVMGO
OBXLAVZ
VWOECDV
WJRJ
IFD
R
F
..-.-.-....-..-.-..-...-.---.--.--...-..-..--.-.----.--.--.--..-...-.---..-.--.-.---.-...-...------.....--..---....----.-----.---........--...-.--..-..-.-.--.----..--...-.-.-...---.-..---..-..-....-.-.----..---.-
40
IKRSTURLRTQNGFLQJYAYUVJIKKJRUSOMB
UXZEJYTM
JHEIADAPLCGO
NPI
CDJRW
NRLER
N
O
X
Q
IKRSTURLRTQNGFLQJYAYUVJIKKJRUSOMBUXZEJYTMJHEIADAPLCGONP
ICDJRWNRLERNOX
Q
IKRSTURLRTQNGFLQJYAYUVJIKKJRUSOMBUXZEJYTM
JHEIADAPLCGONPICDJRWNRLERNOXQ
IKR
STURLRTQNGFLQ
JYAYUVJIKKJRUSOMBUXZEJ
YTMJHEI
ADAPLCGON
PICDJRWNRLE
RNO
X
Q
IKRST
URLRTQNGFLQJYAYUVJIKKJRUSOMBUXZEJYTMJHE
IADAPLC
GONP
ICDJRWN
RLERNOX
Q
IKRSTURLRTQNGFLQJYAYUVJIKKJRUSOMBUXZEJYTMJHEIADAPLCGONPICDJRWNRLERNO
X
Q
IKRSTURLRTQNGFLQJYAYUVJ
IKKJRU
SOMBUXZEJYTMJHEIADAPLCGONPICDJRWNRLERNOX
Q
IKRSTURLRTQNGFLQJYAYUVJIKKJRUSOMBUXZEJYTMJHEI
ADAPLCGONPICDJRWNRL
