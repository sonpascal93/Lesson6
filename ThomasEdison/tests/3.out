57
11641701
0
6
87854627
0
170
101901125
50525666
0
1
191
2
43892065
3530503
0
0
12
5
18
